masonry/src/render_root.rs

```diff
@@ -74,6 +74,7 @@ pub(crate) struct RenderRootState {
     pub(crate) text_layout_context: LayoutContext<BrushIndex>,
     pub(crate) mutate_callbacks: Vec<MutateCallback>,
     pub(crate) is_ime_active: bool,
+    pub(crate) last_sent_ime_area: Rect,
```

Comment:
Maybe there's a better place for this comment, but I think the magic value should be documented.
```suggestion
    /// We ensure that the first valid IME area is sent by using an invalid initial value
    pub(crate) last_sent_ime_area: Rect,
```

Maybe the right solution is to have a:
```rust
/// We ensure that the initial IME area is sent by storing an invalid initial value.
const INITIAL_IME_RECT: Rect = Rect::new(f64::NaN, f64::NaN, f64::NaN, f64::NaN);
```

The comment wording is first-pass and can be refined.

Reply:
Applied with a second pass to the wording.